src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4939,7 +4950,7 @@ public override bool PreVisit(CallNode node)
                 {
                     Contracts.Assert(_currentScope == scopeNew || _currentScope == scopeNew.Parent);
 
-                    if (maybeFunc.AllowsRowScopedParamDelegationExempted(i))
+                    if (maybeFunc.AllowsRowScopedParamDelegationExempted(i, _features))
```

Comment:
>_features [](http://example.com/codeflow?start=76&length=9)

we can remove this. 

Our new binding is similar to row-scope, but should be its own code path because it has enough differences.